### PR TITLE
[News] 2020-11-04

### PR DESCRIPTION
**Pull request by suetsuguk**
- https://github.com/sider/devon_rex/blob/2.27.0/CHANGELOG.md
- https://github.com/sider/runners/blob/0.37.0/CHANGELOG.md

### CODE REVIEW

File: docs/news/2020.md [hunk 1]
```diff
@@ -11,6 +11,67 @@ This document describes notable changes on Sider in the year 2020.
 
 ---
 
+## Update environments and tools on November 2, 2020
```
Comment (ybiquitous) [inline docs/news/2020.md:14]: [suggestion] Can we emphasize Python2 EOL?

```suggestion
## Python2 EOL and environments/tools update
```

Comment (ybiquitous) [reply docs/news/2020.md:14]: `Update environments and tools on ...` is too commonplace, so I'd like to use a different title.

File: docs/news/2020.md [hunk 2]
```diff
@@ -11,6 +11,67 @@ This document describes notable changes on Sider in the year 2020.
...
+
+<time class="news-date" datetime="2020-11-02">November 2, 2020</time>
+
+We have updated the following environments and tools. Please read each tool's documentation for details.
+
+Highlight:
+
+- In the context of [Python 2 end of life](https://www.python.org/doc/sunset-python-2/) on January 1, 2020, we drop the support of Python 2 accordingly.
```
Comment (ybiquitous) [inline docs/news/2020.md:22]: [suggestion] We might be able to use the blog link as follows:

```
As announced [earlier](https://blog.sider.review/drop-the-support-for-python-2-at-the-end-of-october-2020-68d2748545c9), ...
```

File: docs/tools/go/golint.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ hide_title: true
 
 | Language  | Website                        |
 | --------- | ------------------------------ |
-| Go 1.15.2 | https://github.com/golang/lint |
+| Go 1.15.3 | https://github.com/golang/lint |
```
Comment (ybiquitous) [inline docs/tools/go/golint.md:14]: Oh, please don't update the dos for Golint, Govet, and GoMetaLinter.

File: docs/tools/ruby/haml-lint.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ hide_title: true
 
 | Version                   | Language          | Website                          |
 | ------------------------- | ----------------- | -------------------------------- |
-| 0.26.0+ (default: 0.35.0) | HAML (Ruby 2.7.1) | https://github.com/sds/haml-lint |
+| 0.26.0+ (default: 0.36.0) | HAML (Ruby 2.7.2) | https://github.com/sds/haml-lint |
 
 **HAML-Lint** is a static analysis tool to help keep your [HAML](https://github.com/haml/haml) files clean and readable.
 In addition to HAML-specific style and lint checks, it can check them by integrated RuboCop rules.
```
Comment (ybiquitous) [inline docs/tools/ruby/haml-lint.md:15]: Can you please add `parallel` option?

https://github.com/sider/runners/pull/1582

File: docs/news/2020.md (hunk 1 above)
Comment (suetsuguk) [reply docs/news/2020.md]: Fixed. commit c53af1fdcb2b37ffa348c0ef4e0267898069dc37

File: docs/news/2020.md [hunk 5]
```diff
@@ -11,6 +11,68 @@ This document describes notable changes on Sider in the year 2020.
...
+  - [stylelint](../tools/css/stylelint.md) - 13.7.1 → **13.7.2** ([changelog](https://stylelint.io/changelog))
+- C#
+  - [FxCop](../tools/csharp/fxcop.md) - 3.3.0 → **3.3.1** ([release note](https://github.com/dotnet/roslyn-analyzers/releases/tag/v3.3.1))
+- Dockerfile
+  - [hadolint](../tools/dockerfile/hadolint.md) - 1.18.0 → **1.18.2** ([release note](https://github.com/hadolint/hadolint/releases/tag/v1.18.2))
+- Go
+  - [GolangCI-Lint](../tools/go/golangci-lint.md) - 1.31.0 → **1.32.1** ([release note](https://github.com/golangci/golangci-lint/releases/tag/v1.32.1))
+- Java
+  - [Checkstyle](../tools/java/checkstyle.md) - 8.36.1 → **8.36.2** ([release note](https://checkstyle.org/releasenotes.html))
+  - [JavaSee](../tools/java/javasee.md) - 0.1.3 → **0.2.0** ([release note](https://github.com/sider/JavaSee/releases))
```
Comment (ybiquitous) [inline docs/news/2020.md:50]:
```suggestion
  - [JavaSee](../tools/java/javasee.md) - 0.1.3 → **0.2.0** ([release note](https://github.com/sider/JavaSee/releases/tag/0.2.0))
```

File: docs/news/2020.md (hunk 2 above)
Comment (suetsuguk) [reply docs/news/2020.md]: Fixed. commit 4c546032389bd1e3d48d1ac3af5e83098a4957a9

File: docs/news/2020.md [hunk 6]
```diff
@@ -11,6 +11,68 @@ This document describes notable changes on Sider in the year 2020.
 
 ---
 
+## Python2 EOL and environments/tools update
```
Comment (ybiquitous) [inline docs/news/2020.md:14]: [nitpick] Sorry. 🙏 

```suggestion
## Python 2 EOL and environments/tools update
```

File: docs/tools/go/golint.md (hunk 3 above)
Comment (suetsuguk) [reply docs/tools/go/golint.md]: Fixed. commit 200ab01030b6da3db99d3ad1e3f9a6bc6bbbe72a

File: docs/news/2020.md (hunk 5 above)
Comment (suetsuguk) [reply docs/news/2020.md]: Fixed. commit f8884ba703ebecb7722ac953b87a8bb01c4427f7

File: docs/news/2020.md (hunk 6 above)
Comment (suetsuguk) [reply docs/news/2020.md]: Fixed. commit 33f0027c64a1636938e17dc026ed5d860f380380

**Review by yykamei — APPROVED** [review]
LGTM

File: docs/news/2020.md [hunk 7]
```diff
@@ -11,6 +11,68 @@ This document describes notable changes on Sider in the year 2020.
...
+
+<time class="news-date" datetime="2020-11-02">November 2, 2020</time>
+
+We have updated the following environments and tools. Please read each tool's documentation for details.
+
+Highlight:
+
+- As announced [earlier](https://blog.sider.review/drop-the-support-for-python-2-at-the-end-of-october-2020-68d2748545c9), we drop the support for Python 2.
+- As announced [earlier](#recommended-ruleset-for-cpplint), our recommended ruleset for cpplint is now enabled. See our [cpplint document](../tools/cplusplus/cpplint.md#recommended-ruleset) for details.
+- The [`parallel`](../tools/ruby/haml-lint.md#parallel) option of HAML-Lint is newly supported. This option allows you to run linters in parallel.
```
Comment (ybiquitous) [inline docs/news/2020.md:24]: [suggestion] Python 2 drop may be breaking change, so I think it better to have a more helpful description.

Here is my suggestion:

```
In this release, we finally have dropped Python 2 support as [announced earlier](https://blog.sider.review/drop-the-support-for-python-2-at-the-end-of-october-2020-68d2748545c9). If you have not migrated yet, please update your `sider.yml`. (see also [Flake8](../tools/python/flake8.md))

Also, we now have enabled our [recommeded ruleset for cpplint](../tools/cplusplus/cpplint.md#recommended-ruleset) as [announced earlier](#recommended-ruleset-for-cpplint).

And, this release includes the update of the following environments and tools. Please check out each document for details.

Other highlights:

- HAML-Lint supports a new option: [`parallel`](../tools/ruby/haml-lint.md#parallel)

Environments:

...

Tools:

...
```

**Review by ybiquitous — APPROVED** [review]
Thank you. LGTM 👍

**Pull request merged**